src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -101,8 +101,7 @@
         "netstandard1.6": {
             "dependencies": {
                 "NETStandard.Library": "1.6.1",
-                "System.Net.NameResolution": "4.0.0",
-                "System.Diagnostics.StackTrace": "4.3.0"
+                "System.Net.NameResolution": "4.3.0"
```

Comment:
No, I removed it exactly because it's already being brought transitively. It's version is correct.